### PR TITLE
chore(console,toolkit): update custom JWT details page code editor action buttons style

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Update custom JWT details page code editor action buttons style:
1. Add `isLeftMarginIgnored` config for `<CopyToClipboard />` component, all `icon` variant component come with left margin, which is not always wanted.
2. Add `overlay/dark_bg` colors (including `hover`, `pressed` and `focues`)
3. Update action buttons (reset, copy) icon color, gap, and hover/pressed style.

## Testing

Tested locally.
image
image
image
image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]
LGTM, better wait for approval of @charIeszhao

File: packages/console/src/ds-components/CopyToClipboard/index.tsx
Comment (charIeszhao) [inline packages/console/src/ds-components/CopyToClipboard/index.tsx:35]: Can we do this by overriding the className instead of adding a new property?

Comment (darcyYe) [reply packages/console/src/ds-components/CopyToClipboard/index.tsx]: Updated.

**Review by charIeszhao — APPROVED** [review]

**Pull request merged**
